app/bundles/CoreBundle/Tests/Unit/Helper/ExportHelperTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/*
+ * @copyright   2020 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Tests\Unit\Helper;
+
+use Mautic\CoreBundle\Helper\ExportHelper;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class ExportHelperTest extends \PHPUnit\Framework\TestCase
+{
+    /** @var TranslatorInterface */
+    private $translatorInterfaceMock;
+
+    private $dummyData = [
+        [
+            'id'        => 1,
+            'firstname' => 'Mautibot',
+            'lastname'  => 'Mautic',
+            'email'     => 'mautibot@mautic.org',
+        ],
+        [
+            'id'        => 2,
+            'firstname' => 'Demo',
+            'lastname'  => 'Mautic',
+            'email'     => 'demo@mautic.org',
+        ],
+    ];
+
+    protected function setUp(): void
+    {
+        $this->translatorInterfaceMock = $this->createMock(TranslatorInterface::class);
+    }
+
+    /**
+     * Test if exportDataAs() correctly generates a CSV file when we input some array data.
+     */
+    public function testCsvExport()
+    {
+        $helper = $this->getExportHelper();
+        $stream = $helper->exportDataAs($this->dummyData, ExportHelper::EXPORT_TYPE_CSV, 'demo-file.csv');
+
+        $this->assertInstanceOf(StreamedResponse::class, $stream);
+        $this->assertSame(200, $stream->getStatusCode());
+        $this->assertSame(false, $stream->isEmpty());
+
+        ob_start();
+        $stream->sendContent();
+        $content = ob_get_contents();
+        ob_end_clean();
+
+        $csvData = $this->parseCsvString($this->removeBomUtf8($content));
```

Comment:
Is it necessary to parse the CSV string in the test? I would prefer to see the CSV output in the test. What do you think?

Reply:
Fixed in https://github.com/mautic/mautic/pull/9422/commits/b97ba3c0330c5edb3cf5425fc4fdf59e1ce87392, thanks 🚀